WorkflowUI/Sources/ModuleExports.swift

```diff
@@ -0,0 +1 @@
+@_exported import ViewEnvironment
```

Comment:
This is a compatibility gesture, right? Assuming that `Foo` refers to the same module, can two modules call export `Foo` without conflict?

My hunch is that it's fine, and that there's probably a lot of `@_exported Foundation` lurking around, for example, but I figured it worth asking.

Reply:
> This is a compatibility gesture, right?

Yup!

> can two modules call export `Foo` without conflict?

I believe so